docs/conditions.md

```diff
@@ -12,6 +12,8 @@ A simple boolean true/false intended to enable or disable a flag explicitly. The
 FLAGS = {'MY_FLAG': {'boolean': True}}
 ```
 
+The value can be given as a Python `True` or `False` object or as the strings `"true"`, `"True"`, `"False"`, or `"false"`.
```

Comment:
This might be more clear than "object"?

```suggestion
The value can be given as a Python `True` or `False` Boolean value or as the strings `"true"`, `"True"`, `"False"`, or `"false"`.
```

Reply:
Well, but `True` and `False` are objects as well as values. I'll accept it.